internal/service/arcregionswitch/plan_complex_test.go

```diff
@@ -0,0 +1,753 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// lintignore:AWSAT003,AWSAT005,AT004
+package arcregionswitch_test
+
+import (
+	"fmt"
+	"testing"
+
+	sdktypes "github.com/aws/aws-sdk-go-v2/service/arcregionswitch/types"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/names"
+)
+
+func TestAccARCRegionSwitchPlan_route53HealthCheck(t *testing.T) {
```

Comment:
We typically keep all acceptance tests in a single file, even if it winds up being a very long file.